### DISCUSSION TITLE
Rework the Atlantis a bit to be more of a supportive unit

### DISCUSSION THREAD

**Discussion by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#136] [Ideas]
Currently, the factory and air staging part of the Atlantis is rarely used. I think it would be interesting to make this unit more of a supportive unit. Giving it better torpedo defence, tactical missile defence, good sonar, and perhaps Omni when surfaced (is that possible?)

The torpedos could be nerfed a bit, to counteract the buffs

**Comment by PurpleMossClump** [comment]
Wouldn't necessarily need to nerf anything about it, if its felt that it just need to be improved. The UEF have no T3 attack submarine, so the Atlantis having powerful torpedoes is warranted, I feel.
